### PR TITLE
adding timer, members, and fixing some html

### PR DESCRIPTION
**Pull request by barnhartguy**

### CODE REVIEW

File: src/styles/bsidestlv.scss [hunk 1]
```diff
@@ -140,5 +140,35 @@
 a.btn.primary {
   background-color: grey;
   font-size: 3.5rem;
-  border: solid;
+  border: none;
 }
+
+.divTable{
```
Comment (omercnet) [inline src/styles/bsidestlv.scss:146]: If these are only used in CTF page move it to the CTF scss

Comment (barnhartguy) [reply src/styles/bsidestlv.scss:146]: not sure we will only use them there, it depends if we need to build a normal looking div table somewhere else

**Comment by omercnet** [comment]
How about a table scss file to be used where needed?

File: src/styles/bsidestlv.scss (hunk 1 above)
Comment (barnhartguy) [reply src/styles/bsidestlv.scss:146]: we can do that, but we need to add it to the build process (and lookup in the theme)

**Comment by omercnet** [comment]
Yea
Create a table scss file in styles and edit CTF-page.scss to include it

**Pull request merged**